### PR TITLE
Add locators-actions.spec.ts Homework

### PR DESCRIPTION
**Pull request by VasylBurych**
Screenshot_2
Screenshot_3
Screenshot_4
Screenshot_5

### CODE REVIEW

File: tests-examples/demo-todo-app.spec.ts
Comment (ybaranovskyi11) [inline tests-examples/demo-todo-app.spec.ts:1]: you can add this to gitignore or remove

File: tests/practice/locators-actions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import {test, expect, frameLocator, FrameLocator} from "@playwright/test";
```
Comment (ybaranovskyi11) [inline tests/practice/locators-actions.spec.ts]: 'frameLocator' is declared but its value is never read

File: tests/practice/locators-actions.spec.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import {fa, faker} from '@faker-js/faker';
+
+// Test 1 Register form
+test('Register form', async ({ page }) => {
+    await page.goto('https://qa-practice.netlify.app/register');
```
Comment (ybaranovskyi11) [inline tests/practice/locators-actions.spec.ts]: why no POM here?

Comment (VasylBurych) [reply tests/practice/locators-actions.spec.ts:6]: @ybaranovskyi11 we should not integrate POM in this homework (Playwright Locators&Actions HW), but in the next task (Playwright Fixtures HW) we should integrate POM

Comment (VasylBurych) [reply tests/practice/locators-actions.spec.ts]: @ybaranovskyi11 fixed
image

**Review by ybaranovskyi11 — APPROVED** [review]

**Pull request merged**